recipes/snowballstemmer/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.2.1" %}
+{% set sha256 = "919f26a68b2c17a7634da993d91339e288964f93c274f1343e3bbbe2096e1128" %}
+{% set pkg_name = "snowballstemmer" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkg_name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkg_name[0] }}/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - {{ pkg_name }}
+
+about:
+  home: http://snowballstem.org
```

Comment:
PyPI list the repo ( https://github.com/shibukawa/snowball_py ) as the home too.